lib/src/signing.rs

```diff
@@ -233,7 +257,10 @@ impl Signer {
             .find_map(|backend| match backend.verify(data, signature) {
                 Ok(check) if check.status == SigStatus::Unknown => None,
                 Err(SignError::InvalidSignatureFormat) => None,
-                e => Some(e),
+                e => Some(e.map(|mut v| {
+                    v.backend = Some(backend.name().to_owned());
+                    v
+                })),
```

Comment:
nit: Since the backend filed is public, I thought it would now be set by the backend implementation, not by Signer.

Reply:
Ah yeah, of course.
I will look into that (probably later today).